microsoftgraph/microsoft-graph-explorer-v4
# Fix: Shorten overflowing resource links

## Overview
Solves #1320 

### Demo
![image](https://user-images.githubusercontent.com/45680252/151114379-cd533333-38c1-4086-b518-c36582fecfbe.png)

![image](https://user-images.githubusercontent.com/45680252/151114411-e45a7f66-e7d7-497c-865e-c69aebfa5044.png)

![image](https://user-images.githubusercontent.com/45680252/151114450-0066bb0b-da8c-4ba6-ae57-a81cc3d97689.png)

![image](https://user-images.githubusercontent.com/45680252/151114488-a7d72b0c-a92a-4ab0-a953-ce83ee08d4fa.png)


### Notes

## Testing Instructions
- Click on the Resources tab. Navigate through the different links. Notice that for long links, an ellipsis is added at the end. To view the rest of the link, hover over the link or select it.
